lib/awaitableSender.ts

```diff
@@ -187,12 +192,48 @@ export class AwaitableSender extends BaseSender {
           return reject(new OperationTimeoutError(message));
         }, this.sendTimeoutInSeconds * 1000);
 
+        const onAbort = () => {
+          const promise = this.deliveryDispositionMap.get(delivery.id) as PromiseLike;
```

Comment:
null/undefined check for getting from the map is not done in other places where this is used, so didnt want to add a new pattern as part of this PR. If we feel this can potentially cause a problem, a fix in all affected places at once would be preferable

Reply:
Actually, the pattern being used in `if (this.deliveryDispositionMap.has(delivery.id))` and then work with the promise. So using the same in https://github.com/amqp/rhea-promise/pull/66/commits/503399bdb92a16de62b5b62d56144b05e29e0836